src/Chapter1.hs

```diff
@@ -632,7 +642,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n | n < 0  = sumLast2 $ abs n
+           | n < 10 = n
+sumLast2 n = last1 + last2
+ where
+  last1 = mod n 10
+  last2 = div (mod n 100) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that it's possible to rewrite your code to apply `div` and `mod` by the same number to the same argument:

```haskell
mod n 10
div n 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (abs (mod n 100)) 10
```

You can see how we could pattern match on the pair 🙂